package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "jobi",
+  "version": "1.0.0",
+  "description": "Tiny logging utility with events and streams",
+  "keywords": [
+    "logging",
+    "logger",
+    "logs",
+    "log",
+    "jobi",
+    "wood"
+  ],
+  "main": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:StarryInternet/jobi.git"
+  },
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/StarryInternet/jobi/issues"
+  },
+  "homepage": "https://github.com/StarryInternet/jobi",
+  "scripts": {
+    "lint": "eslint .",
+    "mocha": "NODE_ENV=test mocha test/tests --require test/globals.js  --recursive --exit",
+    "nyc": "nyc npm run mocha",
+    "test": "npm run lint && npm run nyc"
+  },
+  "devDependencies": {
+    "chai": "4.2.0",
+    "eslint": "5.16.0",
+    "eslint-config-starry": "8.0.0",
+    "eslint-plugin-mocha": "6.2.2",
+    "eslint-plugin-starry": "6.0.0",
+    "mocha": "6.2.2",
+    "nyc": "14.1.1",
+    "sinon": "8.0.2"
```

Comment:
Can you update all of these to their latest versions?

Also, `eslint-config-starry` requires `eslint-plugin-starry`, so you don't have to pull it in yourself.